Make compatible with Deno 1.0 release.

NathanSMB opened the pull request:
The release files still need to be generated and I'm still investigating why the tests are broken.

NathanSMB committed:
Make compatible with Deno 1.0 release.

fdionisi commented on native/command.rs:
```diff
@@ -1,6 +1,7 @@
 use argon2::{hash_encoded, verify_encoded, Config, ThreadMode, Variant, Version};
 use bytes::Bytes;
-use deno_core::{Buf, CoreOp, ZeroCopyBuf};
+use deno_core::{Buf, Op, ZeroCopyBuf};
+use deno_core::plugin_api::Interface;
```
When working in Rust, I usually prefer to group imports.
```suggestion
use deno_core::{Buf, Op, ZeroCopyBuf, plugin_api::Interface};
```

Sorry about this little obsession 😅

fdionisi reviewed (changes requested):
@NathanSMB thank you very much for this PR.

The code looks great, apart from a little fixation of mine 😊.

Regarding failing tests, I think the issue is that testing libraries are not pinned to a newer version (in both `tests/*.test.ts` files and in the [testing file](https://github.com/fdionisi/deno-argon2/blob/master/tests/testing.test.ts#L1) too).

Also, `Deno.runTests` has been removed (https://github.com/denoland/deno/pull/4922) so the command in the workflow file will throw an error straight away; `tests/run.ts` can be removed and [update the test command](https://github.com/fdionisi/deno-argon2/blob/master/.github/workflows/main.yml#L88) to `deno test --allow-read --allow-write --allow-plugin --allow-run --unstable tests/**/*.test.ts`.

Once this PR will be merged, I'll release a new version so that the native library will be automatically generated 😉.

NathanSMB committed:
Style change

Co-authored-by: Federico Dionisi <fdionisi@users.noreply.github.com>

NathanSMB commented:
No worries about the style change. I haven't touched much rust code before so I was just happy when it compiled and worked!

I still need to look at tests. I will try to take a look at those after work tomorrow but feel free to take a crack at that if you want. I won't be offended lol.

Commit:
Fix test for Deno 1.0

Commit:
Update Deno version in workflow

Commit:
Bring back `tests/run.ts`
Just trying to understand how to succesfully run tests on GitHub Actions
since it fails when running `deno test` w/ a file name pattern.
